### DISCUSSION TITLE
What is the default Room DB storage cache size on Android?

### DISCUSSION THREAD

**Discussion by theta3** [microsoft/cpp_client_telemetry#1029] [General]
Hi all, what is the default size? How do I configure a bigger storage size? Is there an optimal/recommended value?

**Comment by lalitb** [comment]
The default size is documented here - https://github.com/microsoft/cpp_client_telemetry/blob/v3.6.69/docs/Offline-storage-settings.md.
So the default cache size for the file is ~3MB and it's the same for both SQLite and RoomDB, and not overridden by the JNI layer.
